biz.aQute.launcher/src/aQute/launcher/plugin/ProjectLauncherImpl.java

```diff
@@ -161,7 +160,7 @@ private LauncherConstants getConstants(Collection<String> runbundles, boolean ex
 
 		if (!exported && !getNotificationListeners().isEmpty()) {
 			if (listenerComms == null) {
-				listenerComms = new DatagramSocket(new InetSocketAddress(InetAddress.getLocalHost(), 0));
+				listenerComms = new DatagramSocket(new InetSocketAddress("localhost", 0));
```

Comment:
InetAddress.getLoopbackAddress() is since JavaSE-7 whilst the launcher is pointing to ee.j2se-1.6.0.jar. Is it supposed to do that?